### ISSUE TITLE
Need to disable auto-updating C# dependencies

### ISSUE THREAD

**Issue by Greyze** [OmniSharp/omnisharp-vscode#1116]
I work in a secure environment so theres no internet connection, I can only manually transfer files to my machine from another machine with internet.

I've installed VSCode and the C# extension (via downloading the .visx), but the extension downloads other packages on first load which gives an error since theres no internet connection.

So every single time I start VSCode i'm greeted with an output window stating;

```
Updating C# dependencies...
...
Downloading package 'Mono Runtime (Linux / x64)' Failed at stage: downloadPackages
Error: getaddrinfo ENOTFOUND omnisharpdownload.blob.core.windows.net omnisharpdownload.blob.core.windows.net:443
...
```

Is there a way to stop the extension from attempting to update all the time so I don't get greeted with the output window on each load?

I already have the following set;
```
update.channel : none,
extensions.autoUpdate: false
```

Even better, if there's actually an alternative way to install the missing packages I would love to know!

Finally, even though i'm missing some packages I still use the C# extension for syntax highlighting etc.

## Environment data
Platform: Linux, x86_64, Centos 7
VS Code version: 1.8.1
C# Extension version: 1.5.3
Offline machine (Internet connection is impossible)

## Steps to reproduce
Fresh install the C# extension, disable your internet connection.

## Expected  behavior
Stop output window nagging on each fail, or update documentation on how to manually download missing packages.

**Comment by gregg-miskelly** [comment]
I created a wiki page on how to do this: https://github.com/OmniSharp/omnisharp-vscode/wiki/Installing-the-C%23-extension-to-a-computer-without-internet-connectivity

**Comment by Greyze** [comment]
I followed the guide and successfully built the Centos 7 vsix. I've copied it to my offline machine and installed it.

Unfortunately when attempting to debug a dotnet core app (just hello world created by dotnet new); When I press run in VSCode I get the `Error: spawn EACCES`. It then offers to open the launch.json, but I don't see anything wrong. I have of course edited the launch file with the correct program location etc.

Opening VSCodes developer mode I can see;

```
Error: spawn EACCES
    //usr/share/code/resources/app/out/vs/workbench.electron-browser/workbench.main.js:13
at exports._errnoException (util.js:1026:11)
at ChildProcess.spawn (internal/child_process.js:313:11)
at Object.exports.spawn (child_process.js:392:9)
at launchNix (/home/user1/.vscode/extensions/ms-vscode.csharp-1.6.2/out/src/omnisharp/launcher.js:173:35)
at platform_1.PlatformInformation.GetCurrent.then.platformInfo (/home/user1/.vscode/extensions/ms-vscode.csharp-1.6.2/out/src/omnisharp/launcher.js:137:20)
```

...I actually typed that.

**Comment by Greyze** [comment]
Ok I managed to work out the EACCESS error, it was do to with permissions in the extensions folder. I ran this 
`sudo chmod a+x extensions/`
(Of course use the appropriate permissions command in your setup)

I now have my C# apps in Centos compiling and debugging through VSCode, thanks very much for the help.

Just some notes for your guide:

- The tools required stated in the readme.md doesn't mention Python 2.7 being required to run `npm i`.
- Step 7 doesn't work on Windows, when running the gulpfile.js file an exception is thrown at line 95. I commented this out so I could get the Centos 7 vsix to build.
- May be worth mentioning that VSCode on some linux distros may not have permissions to do certain things through extensions
